public/app/features/alerting/unified/RuleList.test.tsx

```diff
@@ -113,11 +113,11 @@ const dataSources = {
 const ui = {
   ruleGroup: byTestId('rule-group'),
   cloudRulesSourceErrors: byTestId('cloud-rulessource-errors'),
-  groupCollapseToggle: byTestId('group-collapse-toggle'),
-  ruleCollapseToggle: byTestId('collapse-toggle'),
+  groupCollapseToggle: byTestId('data-testid group-collapse-toggle'),
```

Comment:
minor nit - can we use the selectors from `@grafana/e2e-selectors` directly here? 🤔 e.g. 
```tsx
import { selectors } from '@grafana/e2e-selectors';
...
groupCollapseToggle: byTestId(selectors.<whatever>),
...
```